src/gilded_rose.spec.js

```diff
@@ -1,10 +1,10 @@
 import { Item, updateQuality } from './gilded_rose';
 
 describe('`updateQuality`', () => {
-  it.todo("Why won't my test pass?", () => {
+  it("Why won't my test pass?", () => {
```

Comment:
Nice job getting this all migrated over! One quick note, though, it looks like this test name might be using our placeholder name.

Reply:
I can see how that looks confusing, just updated that.

Reply:
If this isn't a placeholder test, what do you think about updating the title to indicate what the assertion is checking? Maybe something like this?
```
it('decrements the quality of a haunted shoe', () => {
...
```

Reply:
That looks good. What is the best thing to do, in this situation? I added that one test only to make sure that the `npm test` script was working correctly, thinking that adding more tests and organizing them appropriately could be done in a future step, so I was thinking that the name and the contents of the test weren't really all that important at this point. Would a more appropriate first step to be only to modify the `package.json` and not to worry about modifying the tests?

Reply:
Good question! From that perspective, it sounds like what's actually important here is that no tests fail, instead of having tests pass. If the purpose of this PR is _just_ to setup tests and not actually make any assertions, you can do a couple of things. The first option is to use the `skip` method (`it.skip(...`) which will simply skip this test when running `npm test`. The second option is to leave the existing test using the `.todo` method which will also prevent this step from failing.

Since you said this PR is mainly just to get the test runner setup, is there any reason we need to make modifications to our test suite at all? If we just set up the test script in package.json, we could leave the testing to #6.

Reply:
Sounds good! Sent you a dm about how to go about this in git.